conf/webindex-env.sh.example

```diff
@@ -29,6 +29,15 @@ test -z "$FLUO_HOME" && export FLUO_HOME=/path/to/fluo
 export FLUO_VERSION=`$FLUO_HOME/bin/fluo version`
 export ACCUMULO_VERSION=`accumulo version`
 
+# Accumulo client will likely not work without correct thrift version
+if [ $ACCUMULO_VERSION '<' "1.8" ]; then
```

Comment:
It's doing a lexicographical comparison.  I found an article that discussed this, and now I can't find the same article again.  Not sure why `'<'` is single quoted, I copied that from the article.